user-guide/Reference/faq/General_configuration.md

```diff
@@ -60,17 +60,29 @@ The time server will provide the correct time settings for all DMAs.
 
 To turn any computer into a time server, whether it serves as a DMA or not, do the following.
 
+> [!IMPORTANT]
+> Below information applies to Windows Server 2008 R2, Windows 7 and later.
+
 1. Log on locally to the machine that will act as time server, or establish a remote desktop session with it.
 
-1. Open Registry Editor (*Start \> Run... \> regedit*).
+1. Stop the "Windows Time" service via (*Start \> Control Panel \> System and Security \> Administrative Tools \> Services*).
+
+1. Open the registry editor via (*Start \> Run... \> regedit*).
+
+1. Search for the "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\services\W32Time\TimeProviders\NtpServer".
 
-1. Go to the following key:
+1. Set the *Enabled* value to 1.
 
-   - HKEY_LOCAL_MACHINE/SYSTEM/CurrentControlSet/Services/W32Time/Parameters
+1. Search for "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\services\W32Time\Config".
 
-1. Change the value of LocalNTP from “0” to “1”
+1. Set the *AnnounceFlags* value to 5.
 
-   (0 = No time server except when a domain controller, 1 = Always a time server).
+1. Start the "Windows Time" service again and set the start type to *Automatic*.
+
+> [!NOTE]
+>
+> - When using the Windows server 2012 R2 (64-bit), after Step 7 you have to set the start type to "Automatic (Delayed Start)" for the service to start automatically.
+> - If, when using Windows 7 or the Windows Server 2008 R2, the "Windows Time" service always stops, follow these instructions: [https://support.microsoft.com/en-us/help/2385818](https://support.microsoft.com/en-us/help/2385818).
```

Comment:
This sentence is more complicated than it should be and is a bit hard to read because of it. Adjust to "If you are using Windows 7 or Windows Server 2008 R2, and the Windows Time service always stops, follow the instructions from [this Microsoft support page](https://support.microsoft.com/en-us/help/2385818).

Reply:
done